testdata/p4_16_samples_outputs/pna-add-on-miss.p4.spec

```diff
@@ -61,7 +61,7 @@ action next_hop2 args instanceof next_hop2_arg_t {
 
 action add_on_miss_action2 args none {
 	mov m.MainControlT_tmp 0x0
-	mov m.MainControlT_tmp_0 0x4d2
+	mov m.MainControlT_tmp_0 0x4D2
```

Comment:
I don't know why this change is occurring.
After syncing with main, I regenerated the output file and see this. All hex constants are changed to uppercase.